src/sentry/incidents/endpoints/serializers.py

```diff
@@ -329,6 +331,12 @@ class Meta:
             "resolve_threshold": {"required": False},
         }
 
+    def validate_owner(self, owner_id):
+        # owner_id should be team:id or user:id
+        from sentry.api.fields.actor import Actor
+
+        return Actor.from_actor_identifier(owner_id)
```

Comment:
Do you need to catch errors that come from this and turn them into DRF validation errors?

Reply:
I think I should. 

It looks like `from_actor_identifier` would raise `serializers.ValidationError` but the error message would not be so helpful...so I can catch that and rewrite the message. Do you think I should also catch all errors or just let it fail? I imagine just letting it fail should be okay since we're not just wrapping everything in `try/except Exception` in general

Reply:
Actually, I am thinking of using this function to validate that the owner actually exists before saving / updating it